dspace/config/spring/xmlui/rioxx-xmlui.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans
+        xmlns="http://www.springframework.org/schema/beans"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:util="http://www.springframework.org/schema/util"
+        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+                           http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+    <bean id="rioxxProjectFunderEnricher" class="org.dspace.app.xmlui.objectmanager.RioxxProjectFunderEnricher"/>
```

Comment:
Please inject the "ProjectService" into this bean so that it doesn't have to be retrieved in the Java code (check org.dspace.app.xmlui.objectmanager.RioxxProjectFunderEnricher#getProjectService)